test/utils/config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
We have `config.json` file in another directory. Is it ok to have the same name the file for this one?

Reply:
Yes, might generate some confusion. I will think of a better name.

Reply:
Actually, I think it makes sense to just move this data inside the main config file.